.github/workflows/create-release-pr.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create Release PR
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version you intend to release (eg x.y.z)
+        required: true
+permissions:
+  contents: write
+  pull-requests: write
+env:
+  VERSION: ${{ github.event.inputs.version }}
+jobs:
+  update_version:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.inputs.version != '' }}
+    steps:
+      - name: Validate input version
+        if: ${{ startsWith(github.event.inputs.version, 'v') }}
+        run: |
+          echo "error: version must not start with 'v'."
+          exit 1
+      - name: Checkout repository code
+        uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.19.x'
+      - name: Update docs Version
+        run: make updateversion VERSION=${{env.VERSION}}
+      - name: Create PR
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          BRANCH="release/v${VERSION}"
+          git switch -C ${BRANCH}
+          git add .
+          git commit -m "Update version to ${VERSION}"
+          git push --set-upstream origin --force ${BRANCH} 
+          gh pr create --title "Release v${VERSION}" --body "Release prepared for ${VERSION}"
```

Comment:
We could also add `--notes 'replace me'`, but most of the time it will just get deleted because nothing is changed about this action, except for using a newer version of the CLI